testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/OIDCProtocolMappersTest.java

```diff
@@ -594,6 +594,51 @@ public void testRolesAndAllowedOriginsRemovedFromAccessToken() throws Exception
         }
     }
 
+    /**
+     * Chris Nurse 20-Feb-2022
```

Comment:
No other test case has name and date of the author. Please remove. Date and author will be visible from the Git commit history.